pkg/controller/controller.go

```diff
@@ -908,8 +908,8 @@ func (c *controller) pollInstance(serviceClass *v1alpha1.ServiceClass, servicePl
 		ServiceID: serviceClass.OSBGUID,
 		PlanID:    servicePlan.OSBGUID,
 	}
-	if instance.Status.LastOperation != "" {
-		lastOperationRequest.Operation = instance.Status.LastOperation
+	if instance.Status.LastOperation != nil && *instance.Status.LastOperation != "" {
```

Comment:
Even if you had the isNullOrEmpty you'd have problems with properly logging the value in some places, but yes agreed. toEmptyStringIfNullOrEmpty? :)